pkg/prometheus/operator.go

```diff
@@ -86,6 +86,7 @@ type Config struct {
 	PrometheusConfigReloader     string
 	AlertmanagerDefaultBaseImage string
 	PrometheusDefaultBaseImage   string
+	K8sNamespace                 string
```

Comment:
I was thinking we should just pass in the entire group, instead of "namespace". We will need the namespace as well, but that's for just watching the objects in a single namespace.